package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bandwidth/messaging",
-  "version": "2.0.1",
+  "version": "2.0.3",
```

Comment:
Not sure how 2.0.2 didn't get set on the repo, but 2.0.2 is currently what is on npm so we need 2.0.3 here